### PR TITLE
remove superfluent counter and fix log message

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR fixes (some of) the excessive log sizes caused by #5588 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

File: server/src/main/java/com/cloud/api/ApiSessionListener.java [hunk 1]
```diff
@@ -57,17 +52,15 @@ public void sessionCreated(HttpSessionEvent event) {
         synchronized (this) {
             HttpSession session = event.getSession();
             sessions.put(session, event.getSource());
-            sessionCount++;
         }
-        LOGGER.debug("Sessions count: " + sessions);
+        LOGGER.debug("Sessions count: " + sessions.size());
```
Comment (Pearl1594) [inline server/src/main/java/com/cloud/api/ApiSessionListener.java:56]: can we not use getSessionCount() instead?
```suggestion
        LOGGER.debug("Sessions count: " + getSessionCount());
```

File: server/src/main/java/com/cloud/api/ApiSessionListener.java [hunk 2]
```diff
@@ -57,17 +52,15 @@ public void sessionCreated(HttpSessionEvent event) {
...
     }
     public void sessionDestroyed(HttpSessionEvent event) {
         LOGGER.debug("Session destroyed by Id : " + event.getSession().getId() + " , session: " + event.getSession().toString() + " , source: " + event.getSource().toString() + " , event: " + event.toString());
         synchronized (this) {
-            sessionCount--;
             sessions.remove(event.getSession());
         }
-        LOGGER.debug("Sessions count: " + sessions);
+        LOGGER.debug("Sessions count: " + sessions.size());
```
Comment (Pearl1594) [inline server/src/main/java/com/cloud/api/ApiSessionListener.java:63]: can we not use getSessionCount() instead?
```suggestion
        LOGGER.debug("Sessions count: " + getSessionCount());
```

**Review by rohityadavcloud — APPROVED** [review]

**Review by nvazquez — APPROVED** [review]

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Review by Pearl1594 — APPROVED** [review]

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]

**Pull request merged**